src/general/WorldGenerationSettings.cs

```diff
@@ -1,6 +1,158 @@
-﻿/// <summary>
+﻿using System;
+
+/// <summary>
 ///   Player configurable options for creating the game world
 /// </summary>
 public class WorldGenerationSettings
 {
+    public enum LifeOrigin
+    {
+        Vent,
+        Pond,
+        Panspermia,
+    }
+
+    public enum PatchMapType
+    {
+        Procedural,
+        Classic,
+    }
+
+    /*
+    Values for this particular object
+    */
```

Comment:
```suggestion
    // Values for this particular object
```
This is the comment style used in the code, C-style comments are not used.

Reply:
I used this style because it turned up a few times here, so maybe this should be changed too: https://github.com/Revolutionary-Games/Thrive/blob/master/src/general/OptionsMenu.cs

Reply:
Opened a PR: https://github.com/Revolutionary-Games/Thrive/pull/3422